src/components/PopUpTrackText.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import Popup from 'reactjs-popup';
+import PropTypes from "prop-types";
+import { Button } from 'reactstrap'
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faX } from '@fortawesome/free-solid-svg-icons';
+
+import {
+  Container,
+  CardBody,
+  Card,
+} from 'reactstrap';
+
+export const PopUpTrackText = ({
+  open,
+  text,
+  close,
+}) => {
+    // based off of https://react-popup.elazizi.com/controlled-popup/#using-open-prop
+    return(
+      <div>
+        <Popup open={open} closeOnDocumentClick={false} contentStyle={{width: "760px"}} modal>
+          <Container>
+            <Card>
+              <CardBody style={{boxShadow: "0 4px 8px 0 rgba(0, 0, 0, 0.2)"}}>
+                {/* Close Button */}
+                <Button className="closePopup" onClick={close}><FontAwesomeIcon icon={faX}/></Button>
+                {/* Node info here */}
+                <h4>Object Information</h4>
+                <p>Name: {text}</p>
+              </CardBody>
+            </Card>
+          </Container>
+        </Popup>
+      </div>
+    )
+}
+
+
+export default PopUpTrackText;
+
+PopUpTrackText.propTypes = {
+  open: PropTypes.bool.isRequired,
+  text: PropTypes.string.isRequired,
+  close: PropTypes.func.isRequired,
+}
+
+PopUpTrackText.defaultProps = {
+  open: PropTypes.string.isRequired,
+  close: PropTypes.func.isRequired,
+}
```

Comment:
These are meant to be the default values to use. If a prop's type in `propTypes` has `.isRequired` on it, there's no sense in giving it a default value, since the default can never be used, because the prop will always have a value provided.

And if you do want to set default values, they have to be actual values (so like a boolean of `false` or `true` for `open`, and some function for `close`). They shouldn't themselves be the things in `PropTypes` that we use to name the types of things.

I think we could probably just cut `PopUpTrackText.defaultProps`.
```suggestion
```